releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.0.M6 - December 09, 2019
+
+* Added support for AAD, users can now [Azure TokenCredentials][token_credentials]. Token Credential support allows setting Azure Client ID, Azure Tenant ID, and Azure Client Secret through Environment Variables.
+* Adds support for providing Token Credentials to the provider using the TokenCredentialProvider.
+* Fix Bug where off feature filters would be processed incorrectly resulting in the feature filters still running.
+
+### Breaking Change
+
+* The Role Contributor is no longer used. Two new roles have been added App Configuration Data Owner and App Configuration Data Reader. The member added to the role only needs App Configuration Data Reader for the provider.
```

Comment:
Suggest adding a colon to the second sentence - "Two new roles have been added: App Configuration Data Owner..."